### DISCUSSION TITLE
Is the order of the result guaranteed to match the order of the arguments passed for the search?

### DISCUSSION THREAD

**Discussion by Dmitry-Evsyukov** [milvus-io/milvus#32544] [Q&A and General discussion] [answered]
I found bulk search description in milvus [docs](https://milvus.io/docs/single-vector-search.md#:~:text=the%20calculated%20distances.-,Bulk%2Dvector%20search,-A%20bulk%2Dvector).

But I'm not sure about order of a result. Is the order of the result guaranteed to match the order of the arguments passed for the search? 

I will be glad to receive any answer, thank you!

**Comment by xiaofan-luan** [comment, marked as answer]
Yes, the result is in the same order as search.
